### DISCUSSION TITLE
When installing Vuetify, it's not obvious that icons must be set too

### DISCUSSION THREAD

**Discussion by cadilhac** [vuetifyjs/vuetify#18453] [Q&A]
New to Vuetify, I followed [the instructions in the install section of the docs to add Vuetify to an existing project](https://vuetifyjs.com/en/getting-started/installation/#existing-projects). I then added radio buttons to a template and round buttons are not visible.
After searching on StackOverflow, I tried to add icons to the setup like this:

````
import { createVuetify } from 'vuetify'
import * as components from 'vuetify/components'
import * as directives from 'vuetify/directives'
import { aliases, mdi } from 'vuetify/iconsets/mdi-svg'

const vuetify = createVuetify({
    components,
    directives,
    icons: {
        defaultSet: 'mdi',
        aliases,
        sets: { mdi }
    }
})
````

Why isn't it obvious in the docs (install section, or components needing icons) that this must be added to createVuetify?
Or did I miss something else?

**Comment by carvenl** [comment]
I have encountered the same confusion before, and I suggest adding it to the document.

**Comment by 1Luc1** [comment]
Within the docs link you provided there is this line about icons:

> Lastly, do not forget to install [icons](https://vuetifyjs.com/en/features/icon-fonts/).

Isn't this what you suggest?

**Reply by cadilhac** [reply]
Your're right. It's here. But it's easily missed and you don't know even why it's needed. Maybe a short section instead to show how to install them quickly or at least a mention saying that without them some components would not show, like radio buttons.
Personally when I read this sentence, I said to myself "Oh I don't need icons. I just need basic components". I felt like icons were some fancy graphics, maybe to prefix a button text or these kind of things.
